### PR TITLE
refactor biddingPhase method

### PR DESCRIPTION
**Pull request by seanheinrichs**

### CODE REVIEW

File: src/BiddingFacility.cpp [hunk 1]
```diff
@@ -1,57 +1,66 @@
...
+
+BiddingFacility::~BiddingFacility() {
+    delete bid;
+}
+
+// Gameplay Methods
+
+// TODO: Add verification to ensure that players can afford opening bid (for A1 assume user perfect inputs)
+// Returns the index of the player who goes first (ie. to determine player order)
 int BiddingFacility::biddingPhase(std::vector<Player*> players, int numberOfPlayers) {
```
Comment (mdesson) [inline src/BiddingFacility.cpp:20]: Just to double check, here you want to be passed  a vector of pointers to Players? Or is it supposed to be a pointer to a vector of Players?

File: src/BiddingFacility.h [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #include "Player.h"
 #include <vector>
 
-// TODO: Get rid of the this forward declaration. Current it's only to make the compiler happy.
+// Forward Declaration
```
Comment (mdesson) [inline src/BiddingFacility.h:7]: lol

File: src/BiddingFacilityDriver.cpp [hunk 3]
```diff
@@ -12,28 +12,20 @@ int BiddingFacilityDriver(int argc, char *argv[]) {
...
     std::cin >> numOfPlayers;
-
     std::vector<Player*> players;
 
+    // Create all Players in Player vector
     for(int i = 0; i < numOfPlayers; i++) {
         std::cout << "What is your name?" << std::endl;
         std::cin >> name;
-        std::cout << "What is your date of birth? (YYMMDD)" << std::endl;
+        std::cout << "How many years old are you? (eg 18)" << std::endl;
```
Comment (mdesson) [inline src/BiddingFacilityDriver.cpp]:
```suggestion
        std::cout << "How old are you? (eg 18)" << std::endl;
```

So we sound less like a lizard person trying to sound like a human

**Review by mdesson — CHANGES_REQUESTED** [review]

**Review by cookfisher — APPROVED** [review]

File: src/BiddingFacility.cpp (hunk 1 above)
Comment (seanheinrichs) [reply src/BiddingFacility.cpp:20]: Its a vector that consists of Player pointers (each that point to a unique player object).

File: src/BiddingFacilityDriver.cpp (hunk 3 above)
Comment (seanheinrichs) [reply src/BiddingFacilityDriver.cpp]: LMAO

**Pull request merged**